src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20190715_add_internal_resource_id.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="mbemis" id="add-cluster-internal-resource-id">
+        <addColumn tableName="CLUSTER">
+            <column name="internalId" type="VARCHAR(254)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet logicalFilePath="leonardo" author="mbemis" id="backpopulate-internal-resource-id">
+        <sql stripComments="true">
+            update CLUSTER c set internalId = CONCAT(c.googleProject, '_', c.clusterName);
+        </sql>
+    </changeSet>
+    <changeSet logicalFilePath="leonardo" author="mbemis" id="internal-id-non-nullable">
```

Comment:
Even though it's non-unique I was wondering if we should add an index for `internalId` anyway. However we're never actually using it in a `WHERE` clause (and I don't see why we would), so I think it's ok without one.